airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1336,22 +1340,30 @@ public void testStressPerformance() throws Exception {
     final Map<String, String> fieldFormats = new HashMap<>();
 
     streams.stream().map(AirbyteStream::getJsonSchema).forEach(streamSchema -> {
-      final JsonNode fieldDefinitions = streamSchema.get("properties");
-      final Iterator<Entry<String, JsonNode>> iterator = fieldDefinitions.fields();
-      while (iterator.hasNext()) {
-        Map.Entry<String, JsonNode> entry = iterator.next();
-        if (entry.getValue().has("format")) {
-          String format = entry.getValue().get("format").asText();
-          if (format.equalsIgnoreCase("date") || format.equalsIgnoreCase("date-time")) {
-            fieldFormats.put(entry.getKey(), format);
-          }
-        }
-      }
+      putFieldsFromSchema(streamSchema, fieldFormats);
     });
 
     return fieldFormats;
   }
 
+  private static void putFieldsFromSchema(JsonNode streamSchema, Map<String, String> fieldFormats) {
```

Comment:
nitpick: let's rename this to `findDateTimeFields`

Reply:
done